src/Properties/DateOfBirth.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace GingerPluginSdk\Properties;
+
+use GingerPluginSdk\Bases\BaseField;
+use GingerPluginSdk\Exceptions\OutOfPatternException;
+use GingerPluginSdk\Interfaces\ValidateFieldsInterface;
+
+final class DateOfBirth extends BaseField implements ValidateFieldsInterface
```

Comment:
Could you explain why you extend the basefield. What does it add compared to the earlier implementation?

Reply:
Direct answer on your question : with this extend i already provided it field to toArray method, no other mentions required (except property_name param)